### ISSUE TITLE
Incorrect screen resolution reported

### ISSUE THREAD

**Issue by bradleyjkemp** [hankchizljaw/mybrowser.fyi-project#31]
## Link to mybrowser.fyi report
https://mybrowser.fyi/report/5b637876e3d3580014a03515

## Brief description of what went wrong
Moto G5+ (1080p) screen reports a much smaller screen resolution than is actually correct.

## Steps to reproduce the issue
Open mybrowser.fyi in Chrome on Moto G5+

## Any other notes
Trying other websites gets mixed results:
https://www.infobyip.com/detectscreenresolution.php - Correct

https://visser.io/tools/screen-resolution/ - Incorrect and same values as my browser.fyi

**Comment by Andy-set-studio** [comment]
Hey @bradleyjkemp. Thanks for the report :)

Is the Moto G5+ a 2X style screen like an iPhone?

**Comment by svenluijten** [comment]
@bradleyjkemp what happens when you enable the "Desktop site" option in the Chrome menu? (3 vertical dots in the top right next to the URL).

**Comment by bradleyjkemp** [comment]
I don't believe it's a 2X style screen, it's just a normal android phone, I've never seen any content scaling options.

Desktop mode had no effect on reported resolution: http://mybrowser.fyi/report/5b640c9d34148400145d0eca
